src/stores/account.ts

```diff
@@ -245,4 +250,23 @@ export default class AccountStore implements IAccountStore {
     this._privateKeyToRecovery = null;
     return account;
   };
+
+  fixKeyLength = async (v: ProtectedPrivateKey, passphrase: string) => {
+    const key = V3toRaw(await fs.promises.readFile(v.path, "utf8"), passphrase);
+    if (key.length < 32) {
+      fs.promises.writeFile(
```

Comment:
It would be better to unify the style with `await` instead of callback..

Reply:
Um... I'm kinda not getting it. you mean entire account.ts should be unify style with await, or this function specifically? 

I used `.then()` on some function intentionally (getAccount() for example) to bit more easily use that without any additional async handling.

Reply:
1. 
I think that `async`/`await` style is more helpful than .then() chaining.

> I used .then() on some function intentionally (getAccount() for example) to bit more easily use that without any additional async handling.

If you give an example in which case it is useful to use `then()` chaining, I think we can think together. 🤔 

2. But even if I support the async/await style, I also don't think it's important to fix it all at once considering the size of the PR. it is a comment that does not increase further within the range of change. 😅

Reply:
Oh, my bad. I just assumed getAccount() must be sync function, which is not. I can fix this.